adapters/src/controller/stats.rs

```diff
@@ -13,49 +13,50 @@
 //!
 //! Consider the following interleaving of three threads: transport
 //! endpoint 1, transport endpoint 2, and circuit thread:
-//!
-//! | Thread        | Action                              | `buffered_records` |
-//! actual # of buffered records | | ------------- |
-//! ----------------------------------- | :----------------: |
-//! :--------------------------: | | 1. endpoint 1 | enqueues 10 records
-//! |         10         |              10              | | 2. circuit:   |
-//! `ControllerStats::consume_counters` |         0          |              10
-//! | | 3. endpoint 2 | enqueue 10 records                  |         10
-//! |              20              | | 4. circuit    | circuit.step()
-//! |         10         |              0               |
+// The quotes below are needed because of
+// https://github.com/rust-lang/rustfmt/issues/4210
```

Comment:
I was wondering why the table was illegible. Assumed it showed up nicely in some tools.